src/components/oculus-touch-controls.js

```diff
@@ -0,0 +1,259 @@
+var registerComponent = require('../core/component').registerComponent;
+var bind = require('../utils/bind');
+var trackedControlsUtils = require('../utils/tracked-controls');
+
+const TOUCH_CONTROLLER_MODEL_BASE_URL = 'https://cdn.aframe.io/controllers/oculus/';
```

Comment:
The code base is not ES6. We use just `var` and the uppercase variable name indicates that is a static value.